docs/developer-guide/working-with-rules.md

```diff
@@ -261,6 +261,8 @@ Once you have an instance of `SourceCode`, you can use the methods on it to work
 * `getTokensAfter(nodeOrToken, count)` - returns `count` tokens after the given node or token.
 * `getTokensBefore(nodeOrToken, count)` - returns `count` tokens before the given node or token.
 * `getTokensBetween(node1, node2)` - returns the tokens between two nodes.
+* `getTokenOrCommentAfter(nodeOrToken)` - returns the first token or comment after the given node or token.
```

Comment:
I think we just forgot to update the docs, we didn't intend to keep them private.